### DISCUSSION TITLE
Is it technically feasible to build a decentralized MPC protocol based on CBK?

### DISCUSSION THREAD

**Discussion by johnnynanjiang** [nervosnetwork/ckb#4242] [General]
Given the flexibility and high performance of Nervos blockchain, is it feasible to build a decentralized MPC layer on top of it?

FYI, Lit Protocol and Odsy Netowrk are building decentralized MPC protocols on their own blockchains.

**Comment by xcshuan** [comment]
Lit Protocol and Odsy Netowrk are both some forms of BFT chains and may also rely on trusted hardware, while CKB is a PoW-based permissionless public chain. It is impossible to do such a thing based on CKB nodes.

**Comment by johnnynanjiang** [comment]
Thanks @xcshuan for your clarification, appreciate it.
